packages/desktop-ui-lib/src/lib/settings/plugins/component/plugin-marketplace/+state/actions/plugin-source.action.ts

```
@@ -0,0 +1,30 @@
+		pluginId,
+		versionId,
+		sourceId
+	}));
+	public static restore = createAction('[Plugin Source] Restore', (pluginId: ID, versionId: ID, sourceId) => ({
+		pluginId,
+		versionId,
+		sourceId
+	}));
+	public static selectSource = createAction('[Plugin Source] Select', (source: IPluginSource) => ({ source }));
+	public static reset = createAction('[Plugin Source] Reset');
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)